acceptance/Admin.Specs/Hooks/AdminHooks.cs

```diff
@@ -1,19 +1,34 @@
 using TechTalk.SpecFlow;
 
+using Npgsql;
+
 using Admin.Specs.Drivers;
-using Admin.Specs.PageObjects;
 
 namespace Admin.Specs.Hooks
 {
 
     [Binding]
     public class AdminHooks
     {
-        [BeforeScenario("Admin")]
-        public static void BeforeScenario(BrowserDriver browserDriver)
+        [BeforeScenario]
+        public static async void BeforeScenario(BrowserDriver browserDriver)
         {
-            var rootPageObject = new RootPageObject(browserDriver.Current);
-            // TODO clear login cookies
+            await using var connection = new NpgsqlConnection(
+                "Host=localhost;Database=postgres;Username=postgres;Include Error Detail=true"
+            );
+            await connection.OpenAsync();
+
+            await using
+            (
+                var cmd = new NpgsqlCommand(
```

Comment:
Nice, but would it be worth looking to abstract this behind a service? So it is easy to change the data store?